### PR TITLE
Add a callback that downloads cluster credentials after creation.

### LINKED ISSUE #1517: `az acs create --orchestrator=kubernetes ...` should implicitly perform `get-credentials` afterward
This can be assigned to me.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes #1517

### CODE REVIEW

**Comment by squillace** [comment]
Oh, PLEASE. +24.35

**Comment by lostintangent** [comment]
Excited to see this land!

**Comment by brendandburns** [comment]
@yugangw-msft @derekbekoe @tjprescott can i get a review on this one? its ready to go....

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -410,7 +410,7 @@ def acs_create(resource_group_name, deployment_name, name, ssh_key_value, dns_na
                agent_storage_profile="",
                orchestrator_type="DCOS", service_principal=None, client_secret=None, tags=None,
                windows=False, admin_password="", generate_ssh_keys=False,  # pylint: disable=unused-argument
-               validate=False, no_wait=False):
+               validate=False, no_wait=False, get_credentials=True):
```
[the remark this answers is not in the document]
Comment (brendandburns) [reply src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: Done.

**Comment by brendandburns** [comment]
@yugangw-msft comment addressed, please re-check.

Thanks!
--brendan

**Comment by derekbekoe** [comment]
Closing as this PR has been open for a while and has conflicts.

**Pull request closed without merging**